### DISCUSSION TITLE
Problem when using google drive as a file saver

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6380] [Help]
I'm trying to use google drive as a file saver.

```
                Forms\Components\FileUpload::make('attachment')
                    ->disk('google')

```

Previously I have successfully uploaded to local-storage (public).

But after I try to use google drive,
it looks like the file has been uploaded successfully, but it doesn't work to display again on the web page.

After uploading
![image](https://user-images.githubusercontent.com/6214964/235727827-5e7e4ebc-eff2-432b-8d88-c43399170170.png)

After save button is clicked
![image](https://user-images.githubusercontent.com/6214964/235728050-f8a5f1f6-1c9f-491e-9358-2f2d46a4c9d0.png)

After investigation, the file has been successfully uploaded on Google Drive
and the attachment field has been filled with the file name.
![image](https://user-images.githubusercontent.com/6214964/235728660-028c4902-394f-40eb-9ac6-7df29b747a37.png)

![image](https://user-images.githubusercontent.com/6214964/235728874-dc1aed82-e284-49c5-9474-f63232faf322.png)

I need help to to fix this problem.

some settings in my .env as follows:

APP_URL=http://127.0.0.1:8000

FILESYSTEM_DISK=google

and the package that I use is  "masbug/flysystem-google-drive-ext": "^2.2"

**Comment by danharrin** [comment]
Unfortunately Livewire only works with S3-compatible disks at the moment.

**Comment by diegolisboadev** [comment]
Because FileUpload was unable to view the file in GDrive, however I used the

`Storage::disk('google')->url($record->file)`

and it worked!!
